Client/Frontend/Browser/Onboarding/OnboardingPrivacyConsentView.swift

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+import BraveShared
+import pop
+import SnapKit
+
+extension OnboardingPrivacyConsentViewController {
+    
+    class View: UIView {
+        
+        let yesConsentButton = CommonViews.primaryButton(text: Strings.OBPrivacyConsentYesButton).then {
+            $0.accessibilityIdentifier = "OnboardingPrivacyConsentViewController.YesButton"
+            $0.titleLabel?.adjustsFontSizeToFitWidth = true
+        }
+        
+        let noConsentButton = CommonViews.secondaryButton(text: Strings.OBPrivacyConsentNoButton).then {
+            $0.accessibilityIdentifier = "OnboardingPrivacyConsentViewController.NoButton"
+            $0.titleLabel?.adjustsFontSizeToFitWidth = true
+        }
+        
+        private let mainStackView = UIStackView().then {
+            $0.axis = .vertical
+            $0.distribution = .equalSpacing
+        }
+        
+        private let braveLogo = UIImageView(image: #imageLiteral(resourceName: "browser_lock_popup")).then { logo in
+            logo.contentMode = .scaleAspectFit
+        }
+        
+        private let titleLabel = CommonViews.primaryText(Strings.OBPrivacyConsentTitle).then {
+            $0.font = UIFont.systemFont(ofSize: 20, weight: UIFont.Weight.semibold)
+            $0.numberOfLines = 0
+        }
+        
+        private let copyFromClipboardLabel = UILabel().then {
+            $0.text = "Allow Brave to check my clipboard for a matching referral code"
+            $0.font = .systemFont(ofSize: 16, weight: .medium)
+            $0.numberOfLines = 0
+        }
+        
+        private let refProgramLabel = UILabel().then {
+            $0.text = Strings.OBPrivacyConsentDetail
+            $0.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+            $0.numberOfLines = 0
+            $0.textAlignment = .left
+            $0.minimumScaleFactor = 0.7
+        }
+        
+        init(theme: Theme) {
+            super.init(frame: .zero)
+            
+            mainStackView.tag = OnboardingViewAnimationID.detailsContent.rawValue
+            braveLogo.tag = OnboardingViewAnimationID.background.rawValue
+            
+            applyTheme(theme)
+            
+            mainStackView.addStackViewItems(
+                .view(.spacer(.vertical, amount: 1)),
+                .view(braveLogo),
+                .view(
+                    UIStackView(arrangedSubviews: [titleLabel, refProgramLabel]).then {
+                        $0.axis = .vertical
+                        $0.spacing = 24
+                }),
```

Comment:
Could just do separate `.view` entries with a custom space after `titleLabel` instead of a nested stack view, i.e.

```
.view(titleLabel),
.customSpace(24),
.view(refProgramLabel),
```

Same for below

Reply:
thanks, first time doing it your way with `addStackViewItems`

Reply:
actually it looks worse, because the `mainStackView` distribution is equal centering, and I want to group labels and buttons to be close to each other

compare with the screenshot in PR description
<img width="529" alt="Zrzut ekranu 2020-08-25 o 17 11 07" src="https://user-images.githubusercontent.com/6950387/91192406-2d9f9d80-e6f6-11ea-8862-2303d76fd3dc.png">

Reply:
Ahh I didn't notice that the main stack view had a different distribution. Carry on then :D